engine/storage/image/src/org/apache/cloudstack/storage/image/store/TemplateObject.java

```diff
@@ -299,28 +311,17 @@ public DataTO getTO() {
 
     @Override
     public String getInstallPath() {
-        if (installPath != null)
+        if (installPath != null) {
```

Comment:
Consider using ``Strings.isNullorEmpty`` to cover both the ``null`` and empty cases.